bazel: build failed with `--define with_bthread_tracer=true`

**Describe the bug**

```
In file included from <built-in>:402:
<command line>:3:9: error: macro name must be an identifier
#define -DBRPC_BTHREAD_TRACER 1
        ^
```

``` bash
src/bthread/task_tracer.cpp:26:10: error: 'absl/debugging/stacktrace.h' file not found with <angled> include; use "quotes" instead
#include <absl/debugging/stacktrace.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
         "absl/debugging/stacktrace.h"
src/bthread/task_tracer.cpp:27:10: error: 'absl/debugging/symbolize.h' file not found with <angled> include; use "quotes" instead
#include <absl/debugging/symbolize.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
         "absl/debugging/symbolize.h"
```



**To Reproduce**
```
$ bazel build --define with_bthread_tracer=true :brpc
```

**Expected behavior**


**Versions**
OS: ubuntu 20.04
Compiler: clang-12
brpc: master
protobuf: MODULE.bazel

**Additional context/screenshots**

--- a/.github/workflows/ci-linux.yml
+++ b/.github/workflows/ci-linux.yml
@@ -119,7 +119,17 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v2
-    - run: bazel build --verbose_failures --define with_mesalink=false --define with_glog=true --define with_thrift=true --define with_debug_bthread_sche_safety=true --define with_debug_lock=true --define with_asan=true -- //... -//example/...
+    - run: |
+        bazel build --verbose_failures \
+                    --define with_mesalink=false \
+                    --define with_glog=true \
+                    --define with_thrift=true \
+                    --define with_debug_bthread_sche_safety=true \
+                    --define with_debug_lock=true \
+                    --define with_asan=true \
+                    --define with_bthread_tracer=true \
+                    --define BRPC_WITH_NO_PTHREAD_MUTEX_HOOK=true \
+                    -- //... -//example/...
 
   clang-compile-with-make-protobuf:
     runs-on: ubuntu-22.04
@@ -167,7 +177,18 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v2
-    - run: bazel build --verbose_failures --action_env=CC=clang --define with_mesalink=false --define with_glog=true --define with_thrift=true --define with_debug_bthread_sche_safety=true --define with_debug_lock=true --define with_asan=true -- //... -//example/...
+    - run: |
+        bazel build --verbose_failures \
+                    --action_env=CC=clang \
+                    --define with_mesalink=false \
+                    --define with_glog=true \
+                    --define with_thrift=true \
+                    --define with_debug_bthread_sche_safety=true \
+                    --define with_debug_lock=true \
+                    --define with_asan=true \
+                    --define with_bthread_tracer=true \
+                    --define BRPC_WITH_NO_PTHREAD_MUTEX_HOOK=true \
+                    -- //... -//example/...
 
   clang-unittest:
     runs-on: ubuntu-22.04

--- a/.gitignore
+++ b/.gitignore
@@ -47,3 +47,9 @@ CTestTestfile.cmake
 
 # Ignore protoc-gen-mcpack files
 /protoc-gen-mcpack*/
+
+# Ignore bazel files
+bazel-bin
+bazel-out
+bazel-testlogs
+bazel-brpc

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -405,7 +405,7 @@ cc_library(
     ]),
     defines = [] + select({
         "//bazel/config:brpc_with_bthread_tracer": [
-            "-DBRPC_BTHREAD_TRACER",
+            "BRPC_BTHREAD_TRACER",
         ],
         "//conditions:default": [],
     }),

--- a/src/bthread/task_tracer.cpp
+++ b/src/bthread/task_tracer.cpp
@@ -23,8 +23,8 @@
 #include "butil/fd_utility.h"
 #include "butil/memory/scope_guard.h"
 #include "butil/reloadable_flags.h"
-#include <absl/debugging/stacktrace.h>
-#include <absl/debugging/symbolize.h>
+#include "absl/debugging/stacktrace.h"
+#include "absl/debugging/symbolize.h"
 #include <csignal>
 #include <gflags/gflags.h>
 #include <poll.h>

